x-pack/plugins/security_solution/public/resolver/test_utilities/simulator/index.tsx

```diff
@@ -124,6 +130,20 @@ export class Simulator {
     this.wrapper.setProps({ resolverComponentInstanceID: value });
   }
 
+  /**
+   * Change the indices (updates the React component props.)
+   */
+  public set indices(value: string[]) {
```

Comment:
The Security App loads resolver before it loads the signals index. this lets us simulate that by changing the indices prop on an already mounted simulator.